lib/services/content_store.rb

```diff
@@ -0,0 +1,43 @@
+class Services::ContentStore
+  def hard_refresh_cache
+    finder_paths.map do |base_path|
+      item = content_item(base_path)
+      Rails.cache.write(cache_key(base_path), item)
```

Comment:
These should probably expire at some point in time - even to just make the service self cleaning up. 

The requests from content store will return a time that they can be cached until which would be consistent to acknowledge. That'd also resolve a problem that could happen here if any finder had an update scheduled to be published this would need a manual flush.